dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/ConfigurationController.java

```diff
@@ -399,4 +403,10 @@ public void setCorsWhitelist( @RequestBody String input )
     {
         return config.isReadOnlyMode();
     }
+
+    @RequestMapping( value = "/appHubUrl", method = RequestMethod.GET )
+    public @ResponseBody String getAppHubUrl( Model model, HttpServletRequest request )
+    {
+        return (String) appManager.getAppStoreUrl();
```

Comment:
It seems `AppManager.getAppStoreUrl` returns a String already, do we need to cast to String here?

Reply:
Good point, no we don't I'll remove the cast.